pkg/models/session.go

```diff
@@ -31,3 +31,24 @@ type RecordedSession struct {
 	Width    int       `json:"width" bson:"width,omitempty"`
 	Height   int       `json:"height" bson:"height,omitempty"`
 }
+
+type SessionRecord struct {
+	Record string `json:"record"`
+	Height int    `json:"height"`
+	Width  int    `json:"width"`
+}
+
+type Status struct {
+	Authenticated bool `json:"authenticated"`
+}
+
+type ConfigOptions struct {
```

Comment:
It seems related to config environment ssh service, should not declared in models.